src/desktop/apps/partner2/components/locations/index.styl

```diff
@@ -4,3 +4,8 @@
     vertical-align top
     width calc(100% / 6)
     margin-bottom 20px
+
+.partner2-vat-info
```

Comment:
#minor. Conceptually the VAT info is not tight to any particular location, so might make more sense put it in `contact.styl` below.